modules/ingest-geoip/build.gradle

```diff
@@ -29,10 +29,10 @@ tasks.named('internalClusterTestTestingConventions').configure {
 dependencies {
   implementation('com.maxmind.geoip2:geoip2:3.0.2')
   // geoip2 dependencies:
-  runtimeOnly("com.fasterxml.jackson.core:jackson-annotations:2.13.4")
-  runtimeOnly("com.fasterxml.jackson.core:jackson-databind:2.13.4.2")
-  runtimeOnly("com.fasterxml.jackson.core:jackson-core:2.13.4")
-  implementation('com.maxmind.db:maxmind-db:2.1.0')
+  runtimeOnly("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
```

Comment:
as a matter of fact `com.maxmind.geoip2:geoip2` declares its dependencies as 
```
com.fasterxml.jackson.core » jackson-databind	2.13.4.2
com.fasterxml.jackson.core » jackson-core	2.13.4
com.fasterxml.jackson.core » jackson-annotations	2.13.4
```
so it could be the same argument as in repository-azure? https://github.com/elastic/elasticsearch/pull/91725#discussion_r1026999995

Reply:
I'm okay with us locking it onto the same version. We could always unlock it in the future if need be.

Reply:
This was explicitly unlocked in the past. Upgrading Jackson can sometimes be very difficult. This was decoupled from other uses of Jackson so that we can upgrade them independently. Every time we start using `versions.jackson` again, we recouple them.